code/UMLEditor.py

```diff
@@ -24,10 +24,20 @@
 
 ##########################################################################
 
-# **Write Documentation Here**
-def prompt_exit():
-    print ("To be implemented")
+# Exits out of UMLEditor
+def prompt_exit(model:UMLModel):
+    print ("Do you want to save current working project?")
+    userInput = input().upper()
+
+    # if user wants to save project, call the save_model function to save working project
+    if userInput == 'YES':
+        model.save_model()
```

Comment:
save_model() needs a filename. We might want to ask the user for a filename at this stage.